submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/notifications/templates/ActivationTemplate.kt

```diff
@@ -0,0 +1,14 @@
+package ac.uk.ebi.biostd.notifications.templates
+
+import ebi.ac.uk.notifications.integration.model.NotificationTemplate
+import ebi.ac.uk.notifications.integration.model.NotificationTemplateModel
+
+internal class ActivationTemplate(templateContent: String) : NotificationTemplate<ActivationModel>(templateContent)
+
+internal class ActivationModel(
+    private val mailto: String,
+    private val activationLink: String,
+    private val username: String) : NotificationTemplateModel {
+
+    override fun getParams() = listOf("USERNAME" to username, "URL" to activationLink, "MAILTO" to mailto)
```

Comment:
I think we should set the return value here, is not clear at first glance in my opinion since it's a list of pairs